src/Distributions/Bernoulli/Correctness.dfy

```diff
@@ -32,14 +32,45 @@ module Bernoulli.Correctness {
     reveal Model.Sample();
   }
 
+  // The probability mass function (PMF) of the bernoulli distribution
+  function BernoulliMass(numer: nat, denom: nat): bool -> real
```

Comment:
Here and elsewhere, could we please have functions `opaque` by default?

Reply:
Done